java/core/src/main/java/sleeper/core/properties/table/TableProperty.java

```diff
@@ -299,6 +300,15 @@ static List<TableProperty> getAllInGroup(PropertyGroup group) {
                     "This property isn't currently in use but will be in https://github.com/gchq/sleeper/issues/5870.")
             .propertyGroup(TablePropertyGroup.DATA_STORAGE)
             .build();
+    TableProperty IMPORT_DATA_FROM_EXISTING_TABLE = Index.propertyBuilder("sleeper.table.import.data.from.existing")
+            .defaultProperty(DEFAULT_IMPORT_DATA_FROM_EXISTING_TABLE)
+            .description("This property is used when applying an instance configuration and a table has been added.\n" +
+                    "If this is true, then when adding the table, you expect there to be an existing table with the same table " +
+                    "name that you wish to import data from.\n" +
+                    "If the table does not exist already it will throw a Table Not Found exception.\n" +
+                    "If this is false (default), it will attempt to create a new table")
```

Comment:
I think it's misleading to say you're importing data. You're just reusing the existing table. You're importing the table in the sense that you're adding the existing table back into your configuration.

The capitalisation is odd in "Table Not Found exception". I'm not sure it's quite right to talk about a particular exception here. Usually you won't set a table configuration in a context where you'd get an exception like that. You'll get a failure from the CDK instead.